CONTRIBUTING.md

```diff
@@ -35,3 +35,44 @@ During development two SQLite databases are used:
 
 Both apps will allows you to automatically migrate the database and add seeded development data.
 Please visit both the apps in your browser, I recommend you start with the API app first.
+
+#### Migrations
+
+Database changes are pretty straightforward thanks to EF migrations.
+
+##### Important Note:
+When generating migrations, you can only have the connection string for your desired database platform in the 
+`appsettings.Development.json`.  If you have both, it will either error when attempting the migration, or pretend it did 
+what you wanted and not actually do anything.  You would have to change the code for it to run both migrations at once.
```

Comment:
Add a paragraph about installing EF tools (they don't come by default with the SDK):

```console
$ dotnet tool install dotnet-ef --global
```